### ISSUE TITLE
Chain of queries/mutations

### ISSUE THREAD

**Issue by phungtranminh** [async-graphql/async-graphql#493]
Well this question is probably very controversial.

Basically I would like to do chain several mutations in to a query, and second mutation need result of the first mutation. Would it possible for `async-graphql` to handle?

A separate question though, if I send multiple queries in a query, would `async-graphql` run them in subsequent or in parallel order.

**Comment by bram209** [comment]
This is not part of the spec: 

> Normally the executor can execute the entries in a grouped field set in whatever order it chooses (normally in parallel). Because the resolution of fields other than top‐level mutation fields must always be side effect‐free and idempotent, **the execution order must not affect the result**, and hence the server has the freedom to execute the field entries in whatever order it deems optimal.

see: https://spec.graphql.org/June2018/#sec-Normal-and-Serial-Execution

If your use case requires multiple actions to be executed in a single so-called *"unit of work"*, I would advise you to package those in a single mutation and using a database transaction so you can easily recover if something goes wrong.
